lowStock.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.IO;
+using Azure.Data.Tables;
+using Azure;
+using System.Web;
+
+namespace TableStorage
+{
+
+
+    public class LowStock
+    {
+        string AccountName = Environment.GetEnvironmentVariable("AccountName");
+        string TableFavorite = Environment.GetEnvironmentVariable("TableFavorites");
+
+        string TableItem = Environment.GetEnvironmentVariable("TableItem");
```

Comment:
Same comment re IOptions pattern.

Reply:
This class is missing the IOptions pattern - let me know if you want to talk about how to add it :)